.travis.yml

```diff
@@ -18,6 +15,12 @@ env:
   - RAILS_ENV=test
 
 matrix:
+  matrix:
```

Comment:
This line is superfluous. (Doesn't do anything, so it is harmless.)

Reply:
thanks for spotting that @BanzaiMan - bad cutting and pasting on my part ;)